sdk/release/src/Util.hs

```diff
@@ -249,7 +249,7 @@ deploySourceJarName Artifact{..}
 
 protoSourceJarName :: Artifact a -> Maybe Text
 protoSourceJarName Artifact{..}
-  | Jar Proto <- artReleaseType = Just $ T.replace "_java" "" (snd (splitBazelTarget artTarget)) <> "-speed-src.jar"
+  | Jar Proto <- artReleaseType = Just $ T.replace "_java" "" (snd (splitBazelTarget artTarget)) <> "_java_sources.srcjar"
```

Comment:
Does this change the names of the jars we produce? or is this the logic that finds them

Reply:
This does not change the names of the jars pushed to maven,
this actually handle the changes of the jar name produces by the new proto.bzl.